browser/scripts/ui.js

```diff
@@ -528,13 +517,42 @@ VizorUI.prototype.openPresetSaveDialog = function(serializedGraph) {
 	}
 
 	return presetDialog();
-
 };
 
-VizorUI.prototype.setWorldEditorMode = function(isActive) {
-	return E2.app.toggleWorldEditor(isActive);	// E2.app will emit an event back at us
+VizorUI.prototype.setModeBuild = function() {
+	var state = this.state;
+	state.mode = uiMode.build
+	state.visibility.patch_editor = false
```

Comment:
this should be in the UiStateStore I think, whenever `state.mode` changes, these should also be changed by the Store automatically